src/components/TextBox.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../assets/css/TextBox.css';
+
+
+interface Props {
+  title: string
+  text: string
+  link?: string
+  linkText?: string
+  subtext?: string
+};
+
+export const TextBox = ({link, linkText, title, text, subtext}: Props) => {
+  return (
+    <div className = 'textBox'>
+      <h1 className = 'title'>
+        {title}
+      </h1>
+      <h2 className = 'text'>
+        {text}
+      </h2> 
+      <h3 className = 'subtext'>
```

Comment:
For variables that may be undefined, you should have this structure to indicate that that part of the html will only be shown when defined. It avoids crashes

```
{subtext && (
   <h3 className = 'subtext'>
      {subtext}
      {link && linkText && (
         <a href = {link}>{linkText}</a>
      )}
   </h3>
)}
```